### DISCUSSION TITLE
第6章发布

### DISCUSSION THREAD

**Discussion by kanition** [kanition/pbrtbook#85] [发布更新]
<s>翻译至第6章，施工线在446页。
branch: main commit: cfef671bf81f5ec395adf70ef43989aa70129b85

感谢@HakkerBarry参与翻译6.1节和6.2节部分段落。
↓↓↓展开Assets即可下载

<hr /><em>This discussion was created from the release <a href='https://github.com/kanition/pbrtbook/releases/tag/v0.6.0'>第6章发布</a>.</em></s>

请移步最新发布😉

**Comment by AXS34** [comment]
老大，真心感谢您能够提供如此宝贵的资源，小弟刚开始看您翻译的这本书，我编译后生成的PDF有很多章节的索引和方程的索引都出了问题，变成了两个问号（？？）这样，而且多次编译好像也不能解决，请问您能提供一些解决方法或思路吗，万分感谢！

**Reply by AXS34** [reply]
老大，我已解决这个问题，是我自己冒失，多次编译是可以解决的，可能我昨天第二编译搞错了脚本，谢谢您！

**Reply by kanition** [reply]
谢谢你的支持！最近有别的事忙回复晚了。如果你编译遇到问题，可以选择直接下载已经编译好的版本。点击主页【下载本书】找到pdf即可。
